server/views/partials/breadcrumb.njk

```diff
@@ -0,0 +1,36 @@
+{% from "govuk/components/breadcrumbs/macro.njk" import govukBreadcrumbs %}
```

Comment:
there is now two `breadcrumb.njk` and `breadCrumb.njk` not sure which you wanted to keep?